### PR TITLE
V1.2.1

### PR DESCRIPTION
**Pull request by DarrenWestwood**
Added prompt to save settings after API key field changes #35
Added underpayment slack setting #36

### CODE REVIEW

File: modules/gateways/blockonomics.php [hunk 1]
```diff
@@ -91,6 +98,16 @@ function reqListener () {
 				return false;
 			}
 
+			/**
+			 * Prompt to save changes after setting a new API key 
+			 */
+			var apiKeyField = document.getElementsByName('field[ApiKey]')[0];
+			apiKeyField.onchange = function() {
+				newBtn.disabled = true;
```
Comment (blockonomics) [inline modules/gateways/blockonomics.php]: Is this disabling Test Setup ? When does this btn get enabled again ? 
Prefer not to disable things unless very necessary

**Comment by blockonomics** [comment]
@jusasiiv please review

File: modules/gateways/blockonomics.php (hunk 1 above)
Comment (jusasiiv) [reply modules/gateways/blockonomics.php]: This is functionally the same as showing error message on JS alert() and not allowing to continue
https://screenshots.firefox.com/Z8wUQjhdFmoGFDQ3/92.222.83.165
The Test Setup button will enable once saved/refreshed but will not re-enable if API key field is restored to original value.
Can be left as is or set as other plugins

**Comment by jusasiiv** [comment]
Code reviewed and tested, looks good. Waiting for comment about Test Button disabling, after that ready for merge and release.

File: modules/gateways/blockonomics.php (hunk 1 above)
Comment (blockonomics) [reply modules/gateways/blockonomics.php]: I prefer not to disable the button. Creates UI issues. We can remove this

**Pull request merged**